src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (Object.prototype.hasOwnProperty.call(invertedRobot, robot[key])) {
```

Comment:
The check for property presence is correct, but the object you should be checking is the 'robot' object, not the 'invertedRobot'. You want to ensure that each value in the 'robot' object is unique before assigning it as a key in 'invertedRobot'.